src/pages/blog/release-logs.md

```diff
@@ -0,0 +1,65 @@
+---
```

Comment:
might be nice to rename this to `viewing-logs.md`

Reply:
Done